spec/bundler/friendly_errors_spec.rb

```diff
@@ -66,5 +66,20 @@
 
       expect(Bundler::FriendlyErrors.issues_url(exception)).to eq("https://github.com/bundler/bundler/search?q=First+line+of+the+exception+message&type=Issues")
     end
+
+    it "generates the url without colons" do
+      exception = Exception.new(<<END)
+Errno::EACCES: Permission denied @ dir_s_mkdir - /Users/foo/bar/
+END
+      expect(Bundler::FriendlyErrors.issues_url(exception)).not_to include("%3A")
```

Comment:
Can we include a positive assertion as well? Only testing for negatives/absence is a spec can be misleading/pass for the wrong reasons.